helm/starboard-exporter/templates/vpa.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.verticalPodAutoscaler.enabled -}}
```

Comment:
Should we gate in case HPA or KEDA are also enabled? Either way, I'm fine with this too.

Reply:
In theory, they should be independent, but we might need to in the future depending on how it performs